numaprom/udsink/train_rollout.py

```diff
@@ -58,7 +58,7 @@ def _train_model(uuid, x, model_cfg, trainer_cfg):
     trainer.fit(model, train_dataloaders=DataLoader(dataset, batch_size=64))
 
     LOGGER.debug(
-        "{uuid} - Time taken to train model: {time}} sec",
+        "{uuid} - Time taken to train model: {time} sec",
```

Comment:
Why didn't we catch this in unittest?

Reply:
The test cases catch only INFO level logs.

Reply:
I am gonna refine the test cases for this repo. I just ran quick coverage test and saw alot of lines being not covered